### ISSUE TITLE
Import MAFF pages into wallabag

### ISSUE THREAD

**Issue by tcitworld** [wallabag/wallabag#1107]
http://en.wikipedia.org/wiki/Mozilla_Archive_Format

**Comment by j0k3r** [comment]
I wouldn't put that on the 2.0 milestone, but rather on 2.1. This seems to be a bit complex.. Do you have a example of that kind of archive format which can be used as a test ?

**Comment by tcitworld** [comment]
Oh, yes of course, not 2.0, way further.

You need to install [this extension](https://addons.mozilla.org/en-US/firefox/addon/mozilla-archive-format/) and save a page. Then open the MAFF file as a zip. Here is [the specification](http://maf.mozdev.org/maff-specification.html) and [some examples](http://maf.mozdev.org/maff-test-cases/).

**Comment by j0k3r** [comment]
So you mean people should be able to manually upload a `.maff` in Wallabag?

**Comment by tcitworld** [comment]
Yes, it seems some people currently use this system to save pages, I guess it would be good for them to migrate. Apart from the unzipping part, I guess it should be HTML parsing to find the urls (I didn't have a look yet).
